api/entity/Instrument.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { Musician } from './Musician';
+import { MusicianBand } from './MusicianGroup';
+
+@Entity()
+export class Instrument {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column('varchar', { unique: true })
+  name: string;
+
+  @ManyToMany(() => Musician, (musician) => musician.instruments, {
+    onDelete: 'CASCADE',
+  })
+  musicians: Musician[];
+
+  @ManyToMany(() => MusicianBand, (musicianBand) => musicianBand.instruments, {
+    onDelete: 'CASCADE',
+  })
+  groups: MusicianBand;
+}
```

Comment:
Same reasoning as `Genre.ts`. I don't think this belongs here. We should just remove it and create nested routes.